tests/buyProduct_test.js

```diff
@@ -0,0 +1,17 @@
+const { groupCollapsed } = require("console");
+
+const USER = {
+    email: 'testhfjshfkdsjfhsk@gmail.com',
+    password: 'temp1234',
+
+};
+
+Feature('purchase');
+
+Scenario.only('buy product',  ({ I }) => {
+    I.login(USER);
+    I.amOnPage("http://opencart.qatestlab.net/index.php?route=product/product&product_id=45");
+    I.selectProductAndCheckout();
+    I.wait(5);
```

Comment:
You can use `pause()` to pause at certain moment and view result, but don't save it in repository. Avoid to use static waits in repository.